### PR TITLE
Create operations_test.go

### LINKED ISSUE #501: add unit tests for operations.go
I would like to add unit tests for `internal/config/operations.go`

### PR DESCRIPTION
**Pull request by khareyash05**
**Description**
This PR adds tests for `internal/config/operations.go`
This PR fixes #501 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: internal/config/operations_test.go [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+	if op.Type != int32(meshes.OpCategory_CONFIGURE) {
+		t.Errorf("Expected operation type %v but got %v", int32(meshes.OpCategory_CONFIGURE), op.Type)
+	}
+	if op.Description != "Add-on: SMI Addon" {
+		t.Errorf("Expected operation description %v but got %v", "Add-on: SMI Addon", op.Description)
+	}
+	if len(op.AdditionalProperties) != 1 {
+		t.Errorf("Expected 1 additional properties but got %v properties", len(op.AdditionalProperties))
+	}
+	if op.AdditionalProperties[HelmChartURL] != "https://github.com/linkerd/linkerd-smi/releases/download/v0.1.0/linkerd-smi-0.1.0.tgz" {
```
Comment (Aisuko) [inline internal/config/operations_test.go:132]: In my opinion, I believe we need to update these addresses of service mesh manually. So, it is good we add them to the `config.go` as a variable. And we can use it in both `operation.go` and `operation_test.go` but we should be simple. So, this is not related to current PR.

File: internal/config/operations_test.go [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+	if op.Description != "Add-on: Multi-cluster" {
+		t.Errorf("Expected operation description %v but got %v", "Add-on: Multi-cluster", op.Description)
+	}
+	if len(op.AdditionalProperties) != 3 {
+		t.Errorf("Expected 3 additional properties but got %v properties", len(op.AdditionalProperties))
+	}
+	if op.AdditionalProperties[ServiceName] != "linkerd-gateway" {
+		t.Errorf("Expected %v for %v additional property but got %v", "linkerd-gateway", ServiceName, op.AdditionalProperties[ServiceName])
+	}
+	if op.AdditionalProperties[ServicePatchFile] != "file://templates/oam/patches/service-loadbalancer.json" {
```
Comment (Aisuko) [inline internal/config/operations_test.go:111]: We have a [PR](https://github.com/meshery/meshery-linkerd/pull/495) for removing OAM in the future. So, we do not need to care about the part which was related to OAM. Meshery was drop OAM support already.

**Review by Aisuko — COMMENTED** [review]
Thanks for your contribution. Well done. Please sign-off first.

File: internal/config/operations_test.go (hunk 2 above)
Comment (khareyash05) [reply internal/config/operations_test.go:111]: So Shall I directly remove the OAM lines?

File: internal/config/operations_test.go (hunk 1 above)
Comment (khareyash05) [reply internal/config/operations_test.go:132]: So shall I remove line 132?

File: internal/config/operations_test.go (hunk 2 above)
Comment (Aisuko) [reply internal/config/operations_test.go:111]: Yes, let us remove this.

File: internal/config/operations_test.go (hunk 1 above)
Comment (Aisuko) [reply internal/config/operations_test.go:132]: Keep it in this PR. I believe it is ok. We can merge this first and append another PR for these changes.

**Review by Aisuko — CHANGES_REQUESTED** [review]
Please `signoff` or we cannot merge this PR.

**Comment by khareyash05** [comment]
@Aisuko I signed off my last commit. Does this look ok?

**Comment by khareyash05** [comment]
> Please `signoff` or we cannot merge this and more detail please check here https://github.com/meshery/meshery-linkerd/pull/502/checks?check_run_id=13145299095

@Aisuko I signed off the last commit.is it ok?

**Comment by Aisuko** [comment]
Screenshot 2023-05-02 at 1 58 18 pm

It is still failed

**Comment by khareyash05** [comment]
> Screenshot 2023-05-02 at 1 58 18 pm
> 
> It is still failed

Shall I squash the commits and sign-off it will this solve?

**Comment by Aisuko** [comment]
I believe you can try `git commit --amend --no-edit` and `gti push -f origin khareyash05:test`

**Comment by Aisuko** [comment]
> I believe you can try `git commit --amend --no-edit` and `git push -f origin khareyash05:test`

`git commit -s --amend --no-edit`

**Comment by khareyash05** [comment]
@Aisuko Kindly review #504 for new PR with signed commits
